config/initializers/inflections.rb

```diff
@@ -16,4 +16,5 @@
   inflect.acronym "RAPIDS"
   inflect.acronym "OJT"
   inflect.acronym "RSI"
+  inflect.irregular "is", "are"
```

Comment:
This is cool! I have never had the occasion to use this before (or maybe I have but I didn't know how to do this).

Reply:
I was using pluralize and it wasn't pluralizing "is", so I added this